src/server/pfs/pretty/pretty.go

```diff
@@ -169,28 +171,38 @@ Provenance: {{range .Provenance}} {{.Commit.Repo.Name}}@{{.Commit.ID}} ({{.Branc
 	return nil
 }
 
-// PrintFileInfoHeader prints a file info header.
-func PrintFileInfoHeader(w io.Writer) {
-	fmt.Fprint(w, FileHeader)
+// PrintFileInfoHeader prints a file info header, withCommit indicates whether
+// or not to have fields for the file's commit and when it was committed
+// present.
+func PrintFileInfoHeader(w io.Writer, withCommit bool) {
```

Comment:
This function doesn't appear to ever be used - consider removing it?

Reply:
Yeah good call, I'm going to do an audit, I actually don't think any of these are called.